t/op/taint.t

```diff
@@ -2266,7 +2271,7 @@ foreach my $ord (78, 163, 256) {
 
 {
     use re 'taint';
-    "abc".$TAINT =~ /(.*)/; # make $1 tainted
+    my $x = "abc".$TAINT =~ /(.*)/; # make $1 tainted
```

Comment:
avoid void context

Reply:
Given void context is perfectly sane here, it would be clearer to disable the warning than to bypass it.